libgeopmd/src/DrmSysfsDriver.cpp

```diff
@@ -64,27 +119,52 @@ namespace geopm
     DrmSysfsDriver::DrmSysfsDriver(const PlatformTopo &topo,
                                    const std::string &drm_directory)
         : M_PROPERTIES{SysfsDriver::parse_properties_json(plugin_name(), drm_sysfs_json())}
+        , M_DRM_HWMON_DIR_BY_GEOPM_DOMAIN(map_geopm_index_to_hwmon_path(
+              drm_directory,
+              (topo.num_domain(GEOPM_DOMAIN_GPU) > 0) ? (topo.num_domain(GEOPM_DOMAIN_GPU_CHIP) /
+                                                         topo.num_domain(GEOPM_DOMAIN_GPU))
+                                                      : 0))
         , M_DRM_RESOURCE_BY_GPU_TILE(load_resources_by_gpu_tile(
-                                     drm_directory,
-                                     (topo.num_domain(GEOPM_DOMAIN_GPU) > 0) ? 
-                                     (topo.num_domain(GEOPM_DOMAIN_GPU_CHIP) /
-                                     topo.num_domain(GEOPM_DOMAIN_GPU)) : 0))
-        , m_domain(GEOPM_DOMAIN_GPU_CHIP)
+              drm_directory,
+              (topo.num_domain(GEOPM_DOMAIN_GPU) > 0) ? (topo.num_domain(GEOPM_DOMAIN_GPU_CHIP) /
+                                                         topo.num_domain(GEOPM_DOMAIN_GPU))
+                                                      : 0))
     {
     }
 
-    int DrmSysfsDriver::domain_type(const std::string &) const
+    int DrmSysfsDriver::domain_type(const std::string &name) const
     {
-        return m_domain;
+        // So far, all of the supported i915 DRM signals are tile-scoped and
+        // most of the i915 hwmon signals are card-scoped.
+        if (signal_name_is_from_hwmon(name) && !string_ends_with(name, TILE_SIGNAL_NAME_SUFFIX)) {
```

Comment:
The names of the Helper.hpp apis are kind of generic, I like to add the geopm:: namespace when using, but NBD
```suggestion
        if (signal_name_is_from_hwmon(name) && !geopm::string_ends_with(name, TILE_SIGNAL_NAME_SUFFIX)) {
```